src/components/EventMap.js

```diff
@@ -1,9 +1,12 @@
 import Vue from 'vue';
+import { getFilteredEvents } from 'src/util/events';
+import mapMarker from 'src/assets/images/map_marker.png';
+import mapStyles from 'src/assets/styles/mapbox_styles';
```

Comment:
I've never really been sure where to put styles in the [mapbox specification](https://www.mapbox.com/mapbox-gl-js/style-spec/). I put them in with the other `scss` styles but again, I'm open to other ideas.